### PR TITLE
chore: default page styling + error page refactor

### PR DESCRIPTION
**Pull request by leeuwis**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -1,27 +1,55 @@
 import React from 'react';
 import { NextSeo } from 'next-seo';
-import { Flex, Heading, Text } from 'rebass/styled-components';
+import { Flex, Text, Heading, Box } from 'rebass/styled-components';
+
+const cover = { top: 0, bottom: 0, left: 0, right: 0 };
```
Comment (aidana12) [inline src/pages/index.tsx:5]: Not sure if this should be separated, I don't see the value in that

**Review by aidana12 — CHANGES_REQUESTED** [review]

File: src/pages/_error.tsx [hunk 2]
```diff
@@ -1,47 +1,46 @@
-import { Flex, Text } from 'rebass/styled-components';
+import { NextSeo } from 'next-seo';
+import { Button, Flex, Link, Text } from 'rebass/styled-components';
 
 const getError = ({ res, err }) => {
```
[the remark this answers is not in the document]
Comment (leeuwis) [reply src/pages/_error.tsx:4]: I think the current implementation is the right balance between compact and readable (same how Next does it in their docs)

File: src/pages/index.tsx [hunk 3]
```diff
@@ -1,27 +1,55 @@
...
+        '&:after': {
+          content: "''",
+          position: 'fixed',
+          ...cover,
+          background:
+            'radial-gradient(circle at center bottom, rgba(0,0,0,0.5), rgba(0,0,0,0.8), rgba(0,0,0,0.9), rgba(0,0,0,1))',
+        },
+      }}
+    >
+      {[...Array(5000)].map((_, i) => (
```
[the remark this answers is not in the document]
Comment (leeuwis) [reply src/pages/index.tsx:28]: Was meant to be animated, but the my laptop almost exploded when I tried to animate them 😄 
Feel free to optimize it with an SVG pattern.

**Comment by leeuwis** [comment]
Needs to be reviewed again after the `react-ui` theme update

**Pull request merged**